api/views.py

```diff
@@ -0,0 +1,166 @@
+import django_filters
+from rest_framework import filters
+from rest_framework import generics
+
+from api.models import User, Classifier, Disease, Sample, Mutation
+from genes.models import Gene, Organism
+from api import serializers
+from api.auth import UserUpdateSelfOnly, ClassifierPermission
+
+# Classifier
+
+class ClassifierFilter(filters.FilterSet):
+    created_at__gte = django_filters.IsoDateTimeFilter(name='created_at', lookup_expr='gte')
+    created_at__lte = django_filters.IsoDateTimeFilter(name='created_at', lookup_expr='lte')
+
+    updated_at__gte = django_filters.IsoDateTimeFilter(name='updated_at', lookup_expr='gte')
+    updated_at__lte = django_filters.IsoDateTimeFilter(name='updated_at', lookup_expr='lte')
+
+    class Meta:
+        model = Classifier
+        fields = ['user', 'created_at', 'updated_at']
+
+class ClassifierListCreate(generics.ListCreateAPIView):
+    permission_classes = (ClassifierPermission,)
+    queryset = Classifier.objects.all()
+    serializer_class = serializers.ClassifierSerializer
+    filter_backends = (filters.DjangoFilterBackend,)
+    filter_class = ClassifierFilter
+    ordering_fields = ('user', 'created_at', 'updated_at')
+    ordering = ('created_at',)
+
+class ClassifierRetrieveUpdate(generics.RetrieveUpdateAPIView):
+    permission_classes = (ClassifierPermission,)
+    queryset = Classifier.objects.all()
+    serializer_class = serializers.ClassifierSerializer
+    lookup_field = 'id'
+
+# User
+
+class UserFilter(filters.FilterSet):
+    created_at__gte = django_filters.IsoDateTimeFilter(name='created_at', lookup_expr='gte')
+    created_at__lte = django_filters.IsoDateTimeFilter(name='created_at', lookup_expr='lte')
+
+    updated_at__gte = django_filters.IsoDateTimeFilter(name='updated_at', lookup_expr='gte')
+    updated_at__lte = django_filters.IsoDateTimeFilter(name='updated_at', lookup_expr='lte')
+
+    class Meta:
+        model = User
+        fields = ['email', 'created_at', 'updated_at']
+
+class UserListCreate(generics.ListCreateAPIView):
+    queryset = User.objects.all()
+    serializer_class = serializers.UserSerializer
+    filter_backends = (filters.DjangoFilterBackend,)
+    filter_class = UserFilter
+    ordering_fields = ('created_at', 'updated_at')
+    ordering = ('created_at',)
+
+class UserRetrieveUpdate(generics.RetrieveUpdateAPIView):
+    permission_classes = (UserUpdateSelfOnly,)
+    queryset = User.objects.all()
+    serializer_class = serializers.UserSerializer
+    lookup_field = 'id'
+
+# Genes
+
+class GeneFilter(filters.FilterSet):
```

Comment:
Looks like this provides a django-rest-framework friendly means to access the django-genes api. @rzelayafavila - do you want this ported upstream so that it can be available alongside the tastypie options that you have right now?